examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -146,6 +146,63 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 		}
 	});
 
+	interface UnregisterWebhookRequest extends Request {
+		body: {
+			// The target URL to unsubscribe from change notifications
+			url: string;
+		};
+	}
+	/**
+	 * Unregister's the sender's URL from receiving notifications when the external task-list data changes.
+	 *
+	 * Expected input data format: {@link UnregisterWebhookRequest}
+	 */
+	expressApp.post("/unregister-webhook", (request: UnregisterWebhookRequest, result) => {
+		// 1. Validate request data
+		if (typeof request.body?.url !== "string") {
+			const errorMessage = "Missing or unexpected data in request body";
+			console.log(formatLogMessage(errorMessage));
+			result.status(400).json({ message: errorMessage });
+		} else if (isWebUri(request.body?.url) === undefined) {
+			const errorMessage = "Provided subscription URL is invalid.";
+			console.log(formatLogMessage(errorMessage));
+			result.status(400).json({ message: errorMessage });
+		} else {
+			// 2. Extract SubscriberURL & externalTaskListId from object
+			const subscriberUrl = request.body.url;
+			const externalTaskListId = subscriberUrl.slice(
+				subscriberUrl.indexOf("externalTaskListId=") + "externalTaskListId=".length,
+			);
+
+			// 3. Find cooresponding webook for the given externalTaskListId
+			const webhook = webhookCollection.get(externalTaskListId);
+			if (webhook === undefined) {
+				const errorMessage = "Provided externalTaskListId has no outstanding webhooks";
+				console.log(formatLogMessage(errorMessage));
+				result.status(400).json({ message: errorMessage });
+			} else {
+				// 4. Webhook exists, attempt to remove the subscriber from the webhook
+				if (!webhook.subscribers.includes(subscriberUrl)) {
```

Comment:
Webhook.removeSubscriber performs the same check internally. You could raise the error and simply catch it here instead of repeating the check.

I agree with @alexvy86 suggestion to return a 200 though: https://github.com/microsoft/FluidFramework/pull/17036/files#r1309010338